packages/gatsby/src/services/initialize.ts

```diff
@@ -1,54 +1,52 @@
-/* @flow */
-
-const _ = require(`lodash`)
```

Comment:
I wish github was more contextual and recognize and match these imports :( So much small low hanging fruit there.